auth-module/main.tf

```diff
@@ -1,28 +1,28 @@
-resource "aws_instance" "app-server" {
+resource "aws_instance" "app-server" {                      // zz problem, fix this by enabling kk for a resource with a module path: root->child->grandchild...   
 
   ami           = "ami-09d56f8956ab235b3"
-  instance_type = var.web_instance_type
+  instance_type = var.web_instance_type                     // change this to xx to save yy dollars for a resource with a module path: root->child->grandchild...   
```

Comment:
You have just repeated the same message everywhere. Which means you have only communicated to me what the message format is. What was really required is the actual IDs.
I can't tell if there will be messages in multiple places for a resource. Can't tell which resources the error will be shown on line 1 and for which cases it will be shown on line 4.

Assigh hypothetical Ids to each reasource and in the comment put the resource id as well. Or at the very least like I mentioned in another comment, put actual terraform tree paths.